src/main/java/com/hotjava/app/hotjava/service/VehicleService.java

```diff
@@ -40,7 +40,13 @@ public Vehicle fetchById(int id) {
     @Override
     @CacheEvict(value="vehicle", key="#id")
     public void delete(int id) throws Exception {
-        //vehicleDAO.delete(id);
+        try {
+            //vehicleDAO.delete(id);
+        }
+        catch(Exception e) {
+            e.printStackTrace();
```

Comment:
Placed commented line in a try catch block for when your functionality is placed here. Catches exception and prints its stack trace information.

Reply:
See note above.